test/vm-build

```diff
@@ -44,8 +44,8 @@ def build(machine):
         mock_cmd = ("mock --offline --no-clean --resultdir /var/tmp/mock %s --rebuild /var/tmp/%s"
                     % (mock_opts, os.path.basename(srpm)))
         machine.execute('su - builder -c "%s"' % mock_cmd)
-        machine.download_dir("/var/tmp/mock", "mock")
     finally:
+        machine.download_dir("/var/tmp/mock", "mock")
```

Comment:
We might want to put this into another try/except, since it can also fail, e.g. if the machine isn't responsive. It wouldn't be nice for the script to just crash here.